Setup/docker-compose_dev.yml

```diff
@@ -4,31 +4,54 @@ version: '3.2'
 # YML file for Dev
 
 services:
-    collector:
-      image: securebrowsing/shield-collector:latest
-      volumes:
-        - /var/run/docker.sock:/var/run/docker.sock
-        - /proc:/hostfs/proc:ro
-        - /sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro
-        - /:/hostfs:ro
-      networks:
-        - shield
-      deploy:
-        mode: global
-        restart_policy:
-            condition: any
-        update_config:
-            failure_action: rollback
-        resources:
-          limits:
-            memory: 2GB
-            cpus: "1"
-      environment:
-        - "SYSTEM_PERIOD=5m"
-        - "DOCKER_PERIOD=5m"
+  collector:
+     image: securebrowsing/shield-collector:latest
+     volumes:
+       - /var/run/docker.sock:/var/run/docker.sock
+       - /proc:/hostfs/proc:ro
+       - /sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro
+       - /:/hostfs:ro
+     networks:
+       - shield
+     deploy:
+       mode: global
+       restart_policy:
+           condition: any
+       update_config:
+           failure_action: rollback
+       resources:
+         limits:
+           memory: 2GB
+           cpus: "1"
+     environment:
+       - "SYSTEM_PERIOD=5m"
+       - "DOCKER_PERIOD=5m"
 
-    elk:
-      image: securebrowsing/shield-elk:latest
+  maintence:
+    image: securebrowsing/shield-maintenance:test
```

Comment:
image: securebrowsing/shield-maintenance:latest

Reply:
Need to add to shield-version_dev.txt 
securebrowsing/shield-maintenance:latest image: securebrowsing/shield-maintenance:LABEL